src/main/java/com/questhelper/helpers/quests/thepathofglouphrie/ThePathOfGlouphrie.java

```diff
@@ -0,0 +1,654 @@
+			"Enter the sewer to the west of the Spirit tree");
+		enterSewer.addRequirement(combatGear, prayerPotions, food, crystalChime);
+
+		sewer1Ladder = new ObjectStep(this, ObjectID.LADDER_49700, "Climb up the ladder");
+		sewer2Ladder = new ObjectStep(this, ObjectID.LADDER_49701, new WorldPoint(1529, 4236, 1),
+			"Climb down the ladder.");
+		sewer2Ladder.addRecommended(protectMissiles);
+		sewer3Ladder = new ObjectStep(this, ObjectID.LADDER_49700, new WorldPoint(1529, 4253, 0),
+			"Climb up the ladder.");
+		sewer3Ladder.addRecommended(protectMissiles);
+		sewer4Ladder = new ObjectStep(this, ObjectID.LADDER_49701, new WorldPoint(1486, 4283, 1),
+			"Climb down the ladder. Re-activate your run if you step in any puddles.");
+		sewer4Ladder.addRecommended(protectMissiles);
+		sewer4Ladder.setLinePoints(List.of(
+			new WorldPoint(1530, 4253, 1),
+			new WorldPoint(1530, 4256, 1),
+			new WorldPoint(1512, 4256, 1),
+			new WorldPoint(1512, 4256, 1),
+			new WorldPoint(1512, 4260, 1),
+			new WorldPoint(1512, 4260, 1),
+			new WorldPoint(1510, 4260, 1),
+			new WorldPoint(1510, 4264, 1),
+			new WorldPoint(1503, 4264, 1),
+			new WorldPoint(1503, 4262, 1),
+			new WorldPoint(1496, 4262, 1),
+			new WorldPoint(1496, 4264, 1),
+			new WorldPoint(1484, 4264, 1),
+			new WorldPoint(1484, 4277, 1),
+			new WorldPoint(1482, 4277, 1),
+			new WorldPoint(1482, 4284, 1),
+			new WorldPoint(1486, 4284, 1)
+		));
+		sewer5Ladder = new ObjectStep(this, ObjectID.LADDER_49700, new WorldPoint(1499, 4282, 0),
+			"Climb up the ladder");
+		sewer5Ladder.addRecommended(protectMissiles);
+		bossDoor = new ObjectStep(this, ObjectID.METAL_GATE_49889, new WorldPoint(1506, 4319, 1),
+			"Go to the boss room. Re-activate your run if you step in any puddles.");
+		bossDoor.addRecommended(protectMissiles);
+		bossDoor.setLinePoints(List.of(
+			new WorldPoint(1499, 4283, 1),
+			new WorldPoint(1506, 4283, 1),
+			new WorldPoint(1506, 4291, 1),
+			new WorldPoint(1503, 4291, 1),
+			new WorldPoint(1503, 4292, 1),
+			new WorldPoint(1497, 4292, 1),
+			new WorldPoint(1497, 4291, 1),
+			new WorldPoint(1487, 4291, 1),
+			new WorldPoint(1487, 4302, 1),
+			new WorldPoint(1492, 4302, 1),
+			new WorldPoint(1492, 4304, 1),
+			new WorldPoint(1495, 4304, 1),
+			new WorldPoint(1495, 4316, 1),
+			new WorldPoint(1498, 4316, 1),
+			new WorldPoint(1498, 4319, 1),
+			new WorldPoint(1506, 4319, 1)
+		));
+		// NOTE: If the user logs out, they will be in a non-instanced area of the boss are with the wrong terrorbirds
+		bossStep = new NpcStep(this, new int[]{NpcID.WARPED_TERRORBIRD_12499, NpcID.WARPED_TERRORBIRD_12500, NpcID.WARPED_TERRORBIRD_12501},
+			"Kill the Terrorbirds. You can use the pillars around the room to only fight one at a time. They fight with both Melee and Ranged.");
+		bossStep.setAllowMultipleHighlights(true);
+
+		peekHeavyDoor = new ObjectStep(this, NullObjectID.NULL_49909, WorldPoint.fromRegion(5955, 49, 31, 1), "Peek the heavy door");
```

Comment:
```suggestion
		peekHeavyDoor = new ObjectStep(this, NullObjectID.NULL_49909, WorldPoint.fromRegion(5955, 49, 31, 1), "Peek through the heavy door.");
```

Reply:
Fixed in [`45929d6` (#1289)](https://github.com/Zoinkwiz/quest-helper/pull/1289/commits/45929d6e28d3177d49f095107b19acd08bc68ed4)